index.html

```diff
@@ -30,23 +36,23 @@ <h1 class="primary-text">I'm Mariene Glad to see you!</h1>
                 hestiate to contact me.
             </p>
             <a href="#" class="connect">Let's connect</a>
-        </div>
-        <div class="fa-container">
-            
-              <i class="fa-brands fa-twitter fa"></i>
-              <i class="fa-brands fa-linkedin-in fa"></i>
-              <i class="fa-brands fa-medium fa" ></i>
-              <i class="fa-brands fa-github fa"></i>
-              <i class="fa-solid fa-hand-peace fa"></i>
-           
+            <div class="fa-container">
+                
+                  <i class="fa-brands fa-twitter fa"></i>
+                  <i class="fa-brands fa-linkedin-in fa"></i>
+                  <i class="fa-brands fa-medium fa" ></i>
+                  <i class="fa-brands fa-github fa"></i>
+                  <i class="fa-solid fa-hand-peace fa"></i>
+               
+            </div>
         </div>
     </section>
 
     <section class="works-section">
```

Comment:
While you did address the previous reviewer's requirements, you didn't do so properly. The images for each project card are not sized correctly. Please take a look at the Figma design and fix them.